### PR TITLE
dashboard: When run as root, show URL instead of opening browser

### PR DESCRIPTION
**Pull request by dpandhi-git**
fix in dashboard.go to display dashborad url with root #3558

### CODE REVIEW

File: cmd/minikube/cmd/dashboard.go [hunk 1]
```diff
@@ -128,7 +129,12 @@ var dashboardCmd = &cobra.Command{
 			exit.WithCodeT(exit.Unavailable, "{{.url}} is not accessible: {{.error}}", out.V{"url": url, "error": err})
 		}
 
-		if dashboardURLMode {
+		//check if current user is root
+		user, err := user.Current()
+		if err != nil {
+			exit.WithError("Unable to get current user", err)
+		}
+		if dashboardURLMode && user.Uid == "0" {
```
Comment (tstromberg) [inline cmd/minikube/cmd/dashboard.go]: I think this should be a `||` instead of `&&`, and expect . that this might fail the integration tests as is.

**Comment by tstromberg** [comment]
@minikube-bot OK to test

File: cmd/minikube/cmd/dashboard.go (hunk 1 above)
Comment (dpandhi-git) [reply cmd/minikube/cmd/dashboard.go]: Thanks @tstromberg for the update. I think I focussed too much on root , will update the commit with ||

**Comment by tstromberg** [comment]
Looks good. Thank you!

**Pull request merged**